src/coloring.jl

```diff
@@ -148,7 +148,7 @@ end
 
 function StarSet(star::Dict{Tuple{Int,Int},Int}, hub::Vector{Int}, nb_spokes::Vector{Int})
     # Create a list of spokes for each star, preallocating their sizes based on nb_spokes
-    spokes = [Vector{Int}(undef, ns) for ns in nb_spokes]
+    spokes = Vector{Int}[Vector{Int}(undef, ns) for ns in nb_spokes]
```

Comment:
```suggestion
    spokes = [Vector{Int}(undef, ns) for ns in nb_spokes]
```
the additional annotation is not necessary